src/relay/transforms/fuse_ops.cc

```diff
@@ -1028,7 +1029,7 @@ Pass FuseOps(int fuse_opt_level) {
         auto max_fuse_depth = pc->GetConfig("relay.FuseOps.max_depth", Integer(kMaxFusedOps));
         return Downcast<Function>(FuseOps(f, opt_level, max_fuse_depth.value(), m));
       };
-  return CreateFunctionPass(pass_func, 1, "FuseOps", {"InferType"});
+  return CreateFunctionPass(pass_func, 0, "FuseOps", {"InferType"});
```

Comment:
Shouldn't this be `1` as it's an optimisation in itself?

Reply:
Woops, I forgot to explain this change.
FuseOps is called directly in quite a few places which means it bypasses the 'skip pass' logic in the SequentialPass combinator. As part of this change I ended up just slotting FuseOps in with the rest of the passes, but that broke things since, eg, the VM and Interpreter at least `assume` FuseOps have been run. So it seemed this pass is considered opt level 0 by default.